assets/css/components/tables.scss

```diff
@@ -0,0 +1,37 @@
+table {
+	@extend .govuk-table;
+}
+
+.wp-block-table {
+	display: flex;
+	flex-direction: column-reverse;
+}
+
+.wp-block-table figcaption {
+	@extend .govuk-table__caption;
+	@extend .govuk-table__caption--m;
+}
+
+table thead {
+	border: none !important;
+}
+
+table th, table td {
+	border-top: none !important;
+	border-left: none !important;
+	border-right: none !important;
```

Comment:
**question:** can the use of `!important` be circumvented with a slight bump in specificity e.g. prepending a class to bump by by `10`?

Reply:
Have added main-content class, so both table block and legacy tables are included

Reply:
@sbrody This change prevents visual parity with the Block Editor, as `.main-content` isn't a thing in that context.